resources/tests/models/3340.osm

```diff
@@ -0,0 +1,12463 @@
+
```

Comment:
Can you save these test models with a more descriptive name, including the OS version?